importer/main.py

```diff
@@ -335,10 +335,16 @@ def location_extras(resource, payload_string):
         longitude = "longitude"
 
     try:
-        if locationParentName and locationParentName != "parentName":
-            payload_string = payload_string.replace(
-                "$parentName", locationParentName
-            ).replace("$parentID", locationParentId)
+        if locationParentId and locationParentId != "parentId":
+            payload_string = payload_string.replace("$parentID", locationParentId)
+            if not locationParentName or locationParentName == "parentName":
```

Comment:
Just mentioning here, that this implementation makes the assumption that if only the parentName is provided and the id is not provided, then it's okay to ignore it and remove the whole section

Reply:
Yes, this assumption is correct.